src/evaluation/liveRec/Config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<requirements>
+    <TwoEarsPart sub="src" startup="startBinauralSimulator">binaural-simulator</TwoEarsPart>
+    <TwoEarsPart branch="master"             startup="startAuditoryFrontEnd">auditory-front-end</TwoEarsPart>
```

Comment:
Can we drop the branch requirements?
a) it will constantly change
b) currently, we are not using master of wp2 and wp3.